submissions/ksalpern/Hooli-style-Popup/index.html

```diff
@@ -256,7 +253,7 @@
 
               <li class="nav__item">
                 <a href="#" class="nav__item_bell">
-                  <img class="bell" src="./popup-icons/bell.png" alt="" />
+                  <img class="bell" src="./popup-icons/bell.png" alt="bell" />
```

Comment:
Why do you populate `alt` for this icon and for icons inside popup not?)

Reply:
Do you mean that bell and all images inside popup don't need alt property?